env/env_hdfs.cc

```diff
@@ -124,8 +123,8 @@ class HdfsReadableFile : virtual public SequentialFile,
     Status s;
     ROCKS_LOG_DEBUG(mylog, "[hdfs] HdfsReadableFile preading %s\n",
                     filename_.c_str());
-    ssize_t bytes_read = hdfsPread(fileSys_, hfile_, offset,
-                                   (void*)scratch, (tSize)n);
+    tSize bytes_read =
+        hdfsPread(fileSys_, hfile_, offset, (void*)scratch, (tSize)n);
```

Comment:
Sorry for the last minute comment.
Here, static_cast is preferred.

Reply:
i prefer `tSize` here, return value type from `hdfsPread`, thanks.

Reply:
I mean something like the following.
```
tSize bytes_read = hdfsPread(fileSys_, hfile_, offset, static_cast<void*>(scratch), static_cast<tSize>(n));
```

Reply:
get it, thanks, updated.